### ISSUE TITLE
Redactor 3

### ISSUE THREAD

**Issue by GBH** [comfy/comfortable-mexican-sofa#818]
Right now CMS is using Redactor 1. OEM license was purchased back in 2015 by @nasmorn and donated to the project. This license allows bundling Redactor with open-source projects. So as long as CMS is part of your app, you may use this wysiwyg anywhere in your app.

While Redactor 1 is a great editor, support and updates for it are no longer a thing. Updating to Redactor 2 was not an option as there was no OEM license available and including Redactor with open-source projects was explicitly not allowed.

Redactor 3 once again has OEM license and allows to "Integrate with open source products". See: https://imperavi.com/redactor/buy/

Normally license costs $1799 USD. Since I'm holding old license, it's 50% off at $899 USD. 

If you, or your company interested in integrating Redactor 3 with ComfortableMexicanSofa and willing to cover the cost of the license, let me know.

**Comment by dinatih** [comment]
Hello GBH, what do you think about using a free open-source alternative like https://summernote.org in place of Redactor 3 ?

**Comment by dinatih** [comment]
Yes (you right again, I must think to see how many open issues has a project before propose it 😁).

**Comment by nisevi** [comment]
@GBH by having Redactor 3 are we going to be able to include the plugins listed [here](https://imperavi.com/redactor/plugins/)?

**Comment by GBH** [comment]
Yes, with Redactor 3 I can include all those plugins. Both 1.12 and 2.0 use pretty much the same Redactor functionality so putting Redactor 3.0 in both of them should require very little effort. If there's a demand for it, I don't see why not.

**Comment by dinatih** [comment]
Hello GBH, what do you think about using the new open-source alternative https://trix-editor.org/ in place of Redactor 3 ? :)

**Comment by GBH** [comment]
When Rails 6 comes out and there's an easy way to handle image/file attachments, absolutely.

**Comment by dinatih** [comment]
Yes, ActionText is on version 1 now, coupled with ActiveStorage. It would be awesome 😃

**Comment by gr8bit** [comment]
@dinatih ActionText/trix seems quite limited though, especially when comparing it to Redactor...

@GBH I just re-stumbled upon CMS in search for a _definitive final way_™ to add easy, extensible CMS functionality to our Rails projects. CMS was often recommended, and I just set up a fresh rails project with active storage and CMS in less than 10 minutes and already got a grip on CMS. Impressive work! I'm now wondering about the license of the bundled redactor: If I use CMS in commercial projects (what is exactly what I want to), do I have to have a Redactor license by myself?

**Comment by GBH** [comment]
@gr8bit Redactor bundled with the project already has purchased OEM license that allows you to use it for whatever purpose.

**Comment by gr8bit** [comment]
@GBH Cool, I also just read Redactor's license, sounds totally fitting. Sent you an email :)
